packages/pluggableWidgets/datagrid-web/src/components/Table.tsx

```diff
@@ -67,7 +67,9 @@ export function Table<T>(props: TableProps<T>): ReactElement {
     const [columnOrder, setColumnOrder] = useState<Array<IdType<object>>>([]);
     const [hiddenColumns, setHiddenColumns] = useState<Array<IdType<object>>>(
         (props.columns
-            .map((c, i) => (c.hidable === "hidden" ? i.toString() : undefined))
+            .map((c, i) =>
+                props.columnsHidable && c.hidable === "hidden" && !props.preview ? i.toString() : undefined
```

Comment:
Should this not be:
```suggestion
                props.columnsHidable && c.hidable === "hidden" && props.preview ? i.toString() : undefined
```
?

Reply:
The statement is correct in the original way, it means it will not hide a column if is in preview (Design mode/Studio)